syntax = "proto3";

message PSPushVec
{
    repeated uint64 keys = 1;
    repeated double values = 2;
    int32 timestamp = 3;
}